src/main/java/springsamurais/toyswapbackend/service/imgurapi/service/ImgurService.java

```diff
@@ -0,0 +1,28 @@
+package springsamurais.toyswapbackend.service.imgurapi.service;
+
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import org.springframework.stereotype.Service;
+import springsamurais.toyswapbackend.exception.ImageFailedToUpload;
+
+@Service
+public class ImgurService {
+
+    private final String clientId = "6e3fc450700b556";
+
+    public String uploadImage(String imageBase64, String imageTitle) throws ImageFailedToUpload {
+        HttpResponse<JsonNode> response = Unirest.post("https://api.imgur.com/3/image")
```

Comment:
Nice research and finally find a solution for the image. Good practice to use third party API